samples/test/ssmlAddresses.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const fs = require('fs');
+const {assert} = require('chai');
+
+const ssmlAddresses = require('./../ssmlAddresses');
+
+//const cmd = 'node ssmlAddresses.js';
+const inputFile = 'resources/example.txt';
+const outputFile = 'test-example.mp3';
+
+describe('ssmlAddresses', () => {
+  it('input text tagged with SSML', () => {
+    let expected_ssml = '';
+    try {
+      expected_ssml = fs.readFileSync('resources/example.ssml', 'utf8');
+    } catch (e) {
+      console.log('Error:', e.stack);
+      return;
+    }
+
+    const ssml = ssmlAddresses.textToSsml(inputFile);
+    assert.strictEqual(expected_ssml, ssml);
+  });
+
+  it('synthesize speech to local mp3 file', async () => {
+    let ssml = '';
+    try {
+      ssml = fs.readFileSync('resources/example.ssml', 'utf8');
+    } catch (e) {
+      console.log('Error:', e.stack);
+      return;
+    }
+
+    assert.strictEqual(fs.existsSync(outputFile), false);
```

Comment:
Is testing that the sync exists/before after the function call actually testing the function itself, or is it more testing how your code runs?

Reply:
I was going off of other tests for samples from the repo such as: https://github.com/googleapis/nodejs-text-to-speech/blob/master/samples/test/quickstart.test.js. I changed the test to be consistent with the other tests, but I'll go ahead and change them back to my original test :)

Reply:
Before you change them I'd love a second opinion because I'm unfamiliar with the idiosyncrasies of this repo - @jkwlui I've seen your name on other commits - any strong feelings one way or the other?